src/hansken-js.js

```diff
@@ -2,10 +2,12 @@ import { ProjectContext } from './modules/projectContext.js';
 import { SinglefileContext } from './modules/singefileContext.js';
 import { Scheduler } from './modules/scheduler.js';
 import { SessionManager } from './modules/sessionManager.js';
+import { TraceModelContext } from './modules/traceModelContext.js';
 
 class HanskenClient {
 
     #scheduler;
+    #traceModelContexts = {};
```

Comment:
maybe add a comment here the traceModelContexts is a map of projectId and trace models

Reply:
Added